scipy/stats/_stats_py.py

```diff
@@ -7028,6 +7027,13 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
     The statistic is calculated as ``(mean1 - mean2)/se``, where ``se`` is the
     standard error. Therefore, the statistic will be positive when `mean1` is
     greater than `mean2` and negative when `mean1` is less than `mean2`.
+    
+    This method does not check whether any of the elements of ``std1`` or
+    ``std2`` are negative. If any elements of the ``std1`` or ``std2``
+    parameters are negative in a call to this method, this method will return
+    the same result as if it were passed ``numpy.abs(std1)`` and
+    ``numpy.abs(std2)``, respectively; no exceptions or warnings will be
+    emitted.
```

Comment:
This part looks fine except technically according to the Style Guide, "variables" accepted by the function should be in single backticks (code is double backticks):
```suggestion
    This method does not check whether any of the elements of `std1` or
    `std2` are negative. If any elements of the `std1` or `std2`
    parameters are negative in a call to this method, this method will return
    the same result as if it were passed ``numpy.abs(std1)`` and
    ``numpy.abs(std2)``, respectively; no exceptions or warnings will be
    emitted.
```
The rest looks unintentional?

Reply:
> The rest looks unintentional?

I'm not sure what you're referring to when you say "the rest". Could you clarify?

Reply:
Ah, the wrapping. Must be editor settings. I'll undo those changes.

Reply:
Removed the double backticks, as suggested, in https://github.com/scipy/scipy/commit/8b60b333ccf2df714169909440e9a8273395f5e9.